service/history/tests/const.go

```diff
@@ -0,0 +1,125 @@
+// The MIT License
```

Comment:
We have `defs.go` in many places already. What to do you think about naming this file in the same way?

Reply:
this is not really definitions, but vars for tests. i will rename it to vars.go for this PR

Reply:
ie "definition of var for tests". It depends on how you think about it. I don't have strong opinion on file names. But I have strong opinion on consistency of those names. Let's make a call, figure out the name for such code (almost every package has it) and use it everywhere.

Reply:
> But I have strong opinion on consistency of those names.

i believe we should focus on things that matters? 
file names does not participate in imports (dir name do, but that is a different thing)
as long as file name itself gives an indication of what it contains, that should do the work